isis/tests/FunctionalTestsSocetLineScanKeywords.cpp

```diff
@@ -0,0 +1,153 @@
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Histogram.h"
+
+#include "socetlinescankeywords.h"
+
+#include "TestUtilities.h"
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+
+using namespace Isis;
+using namespace testing; 
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/socetlinescankeywords.xml").expanded();
+
+TEST_F(MroCube, FunctionalTestSocetLineScanKeywordsHirise) {
+  setInstrument("-74999", "HIRISE", "MARS RECONNAISSANCE ORBITER");
+  
+  QTemporaryDir prefix;
+  testCube->reopen("rw");  
+  QString outFileName ="/tmp/outTEMP.txt";
```

Comment:
Remove prefix variable and use the tempDir from the fixture in the outFileName. Looks like this needs to happen in all tests, unless there is something I am missing.